ibis/pyspark/tests/test_timecontext.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+
+pytest.importorskip('pyspark')
+pytestmark = pytest.mark.pyspark
+
+
+def test_table_with_timecontext(client):
+    table = client.table('time_indexed_table')
+    context = (pd.Timestamp('20170102'), pd.Timestamp('20170103'))
```

Comment:
Do these timestamp refer to time in UTC or local timezone?

Reply:
UTC, I meant to add the fix to specify timezone in later PR actually